src/hooks/useBunzz.ts

```diff
@@ -0,0 +1,201 @@
+import bunzz, { Contract } from "bunzz-sdk";
+import { ethers } from "ethers";
+import { useEffect, useState } from "react";
+import NFTAbi from "assets/abis/nft.json";
+import { toast } from "react-toastify";
+
+export const chainId = 80001;
+const DAPP_ID = process.env.REACT_APP_DAPP_ID || "";
+const API_KEY = process.env.REACT_APP_API_KEY || "";
+const IPFS_GW_BASEURL_JSON = `https://cloudflare-ipfs.com/ipfs`;
+const CONTRACT_ADDRESS = process.env.REACT_APP_CONTRACT_ADDRESS || "";
+const MODULE_NAME = "NFTMintingModule";
+const NETWORK_INFO = {
```

Comment:
Should it be fixed? Or better to make it in a setup?🤔

Reply:
These are customizable values by user:)
If it changes for each environment, such as dev or prod, it can be changed to environment variables.
But that's up to the user.

Reply:
@show-coco 
I see. Then, could you please leave a comment here for users?🙏

Reply:
@NailKhalimov I added comment! Please check it.
I will explain for each values in document. Don't worry.